recipe/meta.yaml

```
@@ -10,34 +10,48 @@ source:
   sha256: fe859b0e98d222b4b458022007daac2757e970f0671d45e55d01931c1b4bb3cb
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv "
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cffi                                   # [build_platform != target_platform]
+    - pip
     - {{ compiler('c') }}  # [not (win and py27)]
   host:
-    - cffi >=1.0.0
-    - pip
+    - cffi =>1.15.0
```

```suggestion
    - cffi 1.15
```
Exact pinned are preferred in host for c libs, so that we don't end up rebuilding later with a different version.